scripts/scil_tractogram_alter.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
```

Comment:
I would indeed rename. But not just bundle: the goal of this script is not simply to alter. I would imagine that a scil_tractogram_alter.py would ask user *how* to alter. Here it decides alone (if no option is provided). Can we put a longer, but more significative name? scil_bundle_alter_to_dice? 

Also, I see that in your new methods (ex, subsample_streamlines_alter), you have an option `baseline_sft=None` that is never used in your script. Can you add an option `--reference_bundle` in your script?

Reply:
I am not sure what you mean by "Here it decides alone (if no option is provided)."

I agree that the name  scil_bundle_alter.py does not mean much
Here are a few more suggestions:
- scil_bundle_alter_preserve_overlap.py
- scil_bundle_alter_optimize_overlap.py
- scil_bundle_alter_to_target_dice.py

(alter could be replaced by adjust, vary, adapt, modify, but I like alter because of the definition: change or cause to change in character or composition, typically in a **comparatively small but significant way**)

baseline_sft: It is used in another function (replace)